From 5e9eff188ada13e1c1986d5611a42e0779fe7fd1 Mon Sep 17 00:00:00 2001
From: Thao Nguyen <thao.nguyen.yb@renesas.com>
Date: Mon, 18 Sep 2017 17:09:21 +0700
Subject: [PATCH] Fix QA issue when build with Yocto

Yocto project has updated Linker Hash Style Changed in YP2.4.
Makefile should be updated to avoid QA error.

ERROR: open-avb-1.1+gitAUTOINC+5e43fa5ae2-r0 do_package_qa: QA Issue: No GNU_HASH in the elf binary: '/proj/yocto/gen3/v3.4.0-pt2/build/tmp/work/aarch64-poky-linux/open-avb/1.1+gitAUTOINC+5e43fa5ae2-r0/packages-split/open-avb-gptp/usr/bin/daemon_cl' [ldflags]
ERROR: open-avb-1.1+gitAUTOINC+5e43fa5ae2-r0 do_package_qa: QA run found fatal errors. Please consider fixing them.
ERROR: open-avb-1.1+gitAUTOINC+5e43fa5ae2-r0 do_package_qa: Function failed: do_package_qa

Signed-off-by: Thao Nguyen <thao.nguyen.yb@renesas.com>
Signed-off-by: Masaru Nagai <masaru.nagai.vx@renesas.com>
---
 daemons/gptp/linux/build/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemons/gptp/linux/build/Makefile b/daemons/gptp/linux/build/Makefile
index 1e00b1a..99fbf84 100644
--- a/daemons/gptp/linux/build/Makefile
+++ b/daemons/gptp/linux/build/Makefile
@@ -131,7 +131,7 @@ LDFLAGS_G += -lpthread -lrt
 
 CFLAGS = $(CFLAGS_G)
 CPPFLAGS = $(CPPFLAGS_G)
-LDFLAGS = $(LDFLAGS_G)
+LDFLAGS += $(LDFLAGS_G)
 
 all: $(OBJ_DIR)/daemon_cl
 
-- 
1.9.1

